sqlglot/schema.py

```diff
@@ -121,6 +171,24 @@ def ensure_schema(schema):
     return MappingSchema(schema)
 
 
+def ensure_column_mapping(mapping):
+    from sqlglot.dataframe.sql import types as df_types
```

Comment:
This dependency is a smell IMO.

Should the dataframe API have its own schema?

Reply:
Yeah going that direction now: https://github.com/tobymao/sqlglot/pull/568#discussion_r992944368